xml/depl_maintenance.xml

```diff
@@ -2181,4 +2181,30 @@ migrate</screen>
      </step>
   </procedure>
  </sect1>
+ <sect1 xml:id="sec-depl-maintenance-openstack-tasks">
+  <title>Periodic OpenStack Maintenance Tasks</title>
+  <para>
+    Heat-manage helps manage Heat specific database operations. The associated
+    database should be periodically purged to save space. The following should
+    be setup as a cron job at
+    <literal>/etc/cron.weekly/local-cleanup-heat</literal>
+    with the following content:
+  </para>
+  <screen>
+     #!/bin/bash
+     su heat -s /bin/bash -c "/usr/bin/heat-manage purge_deleted -g days 14" || :
```

Comment:
Instead of `|| :` might it be better to use the less cryptic `|| true`?

Not sure where the reference example job definition came from, and it is perfectly valid. Just a thought...

Reply:
these came from the scripts that Bernhard runs (presumably on the ECP) and are quoted directly in the ticket, so I didnt want to mess with them